### ISSUE TITLE
Support abbreviated and long parameters for PubID parsing

### ISSUE THREAD

**Issue by mico** [metanorma/nist-pubid#14]
While working on integration to relaton-nist I found it's using PubID with additional parameters in long representation: 
`bib = RelatonNist::NistBibliography.get "NIST IR 8011 Vol. 4"`
Along with short version:
`bib = RelatonNist::NistBibliography.get "NIST IR 8011v4"`

Looking at proposal (https://www.nist.gov/system/files/documents/2021/08/26/Publication-ID-Proposal_26Aug21.pdf), I found there is also different options how authors could refer to publications:

> • NIST SP 800-53, Revision 4, includes updates as of 01-22-2015
> • NIST SP 800-53 (April 2013), updated January 22, 2015
> • NIST SP 800-53, Rev. 4, updated 01-22-2015

They are references to the same document.

Now we only parsing using short notation (`NIST SP 800-53r4`).

Should we add support to parse using abbreviated and long PubID parameters?

**Comment by ronaldtse** [comment]
> Should we add support to parse using abbreviated and long PubID parameters?

Yes, this would be good, in order to support parsing of legacy formatted titles so that we can convert them into proper NIST PubID.
